db/migrate/20180924172946_devise_create_users.rb

```diff
@@ -32,8 +32,8 @@ def change
       # t.string   :unlock_token # Only if unlock strategy is :email or :both
       # t.datetime :locked_at
 
-
-      t.timestamps null: false
+      t.json :tokens
```

Comment:
:( 
You did it very well in all over PR, please follow as you did before. Do not edit a mirgation from a PR to another, add a migration if you have something to modify.

Reply:
Ok, I'm gonna undo the changes I made in the last commit.

Reply:
I made a new migration to change those parameters.